### PR TITLE
CY-145 - Added Cloudify Console advanced settings documentation

### PR DESCRIPTION
**Pull request by qooban**

### CODE REVIEW

File: content/manager_webui/overview.md [hunk 1]
```diff
@@ -61,3 +61,11 @@ The UI templates are the sets of pages presented to the users upon logging into
 - Local Blueprint Catalog
 - Deployments
 - System Resources
+
+## Advanced Configuration
+
+In `/opt/cloudify-stage/dist/userData/userConfig.json` on Cloudify Manager machine you can perform more advanced customization of Cloudify Console.
```
Comment (bmansheim) [inline content/manager_webui/overview.md]: Avoid "perform" unless you want to sing
perform > configure

File: content/manager_webui/overview.md [hunk 2]
```diff
@@ -61,3 +61,11 @@ The UI templates are the sets of pages presented to the users upon logging into
...
+
+List of possible settings and its default values can be found in [cloudify-cosmo/cloudify-stage repository](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/conf/userConfig.json).
```
Comment (bmansheim) [inline content/manager_webui/overview.md]: List of possible settings and its default values can be found in >
You can see a list of advanced settings and values in

File: content/manager_webui/overview.md (hunk 1 above)
Comment (qooban) [reply content/manager_webui/overview.md]: Done. BTW - there are two similar usages of `perform` in this doc. One is yours ;-)

File: content/manager_webui/overview.md (hunk 2 above)
Comment (qooban) [reply content/manager_webui/overview.md]: Done.

**Pull request merged**